tests/acceptance/30_generic_methods/file_copy_from_local_source_recursion.cf

```diff
@@ -52,9 +52,10 @@ bundle agent test
 bundle agent check
 {
   vars:
-    "permissions_test_mode"  string => "/usr/bin/test $(/usr/bin/stat -c %a ${init.destination_file}) = \"${init.mode}\"";
-    "permissions_test_owner" string => "/usr/bin/test $(/usr/bin/stat -c %U ${init.destination_file}) = \"${init.owner}\"";
-    "permissions_test_group" string => "/usr/bin/test $(/usr/bin/stat -c %G ${init.destination_file}) = \"${init.group}\"";
+    "owner_id"                  int => getuid("${init.owner}");
```

Comment:
Ah actually I meant to do the opposite: never use 0 for the owner, because it's not necessary, and "root" is more well known. I'll homogenise these.